manifests/garden/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
Can we just use a `NodePort` service instead?

Reply:
Any particular reason to consider `NodePort` ?

Reply:
It seems odd to use an `ingress` for development when we don't even use one in production

Reply:
The problem I was trying to fix with this is exposing the wego-api to be accessible locally. I'm curious if there is any particular advantage on using `NodePort` over `Ingress` ?

Reply:
I had another thought. I think using a `NodePort` service would involve a new service. But I'm trying to use the same service we already have and avoid having an extra one just for this.

Reply:
@jpellizzari  Something I would like to clarify here is if we want to expose the UI as well or just the server like `local-dev` target does?. I understood we should expose the UI as well. But I could be wrong. I was trying to avoid running two different commands to access the UI. Or if we just need to access the UI to work in backend modifications.